Tribler/Core/Modules/MetadataStore/OrmBindings/channel_metadata.py

```diff
@@ -0,0 +1,259 @@
+import os
+from datetime import datetime
+
+from libtorrent import file_storage, add_files, create_torrent, set_piece_hashes, bencode, torrent_info
+from pony import orm
+from pony.orm import db_session
+
+from Tribler.Core.Modules.MetadataStore.serialization import MetadataTypes, ChannelMetadataPayload
+from Tribler.Core.exceptions import DuplicateTorrentFileError, DuplicateChannelNameError
+
+CHANNEL_DIR_NAME_LENGTH = 60  # Its not 40 so it could be distinguished from infohash
+BLOB_EXTENSION = '.mdblob'
+
+
+def create_torrent_from_dir(directory, torrent_filename):
+    fs = file_storage()
+    add_files(fs, directory)
+    t = create_torrent(fs)
+    # For a torrent created with flags=19 with 200+ small files
+    # libtorrent client_test can't see its files on disk.
+    # optimize_alignment + merkle + mutable_torrent_support = 19
```

Comment:
Does this mean that piece alignment is disabled for the first version?

Reply:
yes. We are append-only bulk storage anyways, so it makes no sense now, especially with that problem of 200+ small files.